src/services/muutospyynnot/muutospyyntoUtil.js

```diff
@@ -397,3 +397,13 @@ export function setAttachmentUuids(
     );
   }, changeObjects);
 }
+
+// Combine arrays recursively
+export function combineArrays(arrays) {
+  if (arrays && arrays.length > 0) {
+    const currentArray = arrays.pop();
+    if (currentArray) {
+      return currentArray.concat(combineArrays(arrays));
+    } else return [];
+  } else return [];
```

Comment:
`else` keywords aren't needed here, right? But it's ok like this.

Reply:
Better to feel save than sorry :)